examples/gallery/gallery.60

```diff
@@ -127,6 +128,27 @@ App := Window {
                     }
                 }
             }
+            Tab {
+                title: "TextEdit";
+                VerticalLayout {
+                    GroupBox {
+                        title: "Word-Wrap";
+                        te1 := TextEdit {
+                            text: "Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum";
```

Comment:
I suggest to replace this text with something more concrete, but I can also create a PR later :-)